Package.swift

```diff
@@ -14,8 +14,8 @@ let package = Package(
     )
   ],
   dependencies: [
-    .package(url: "https://github.com/pubnub/kmp-chat", exact: "0.8.2-dev"),
-    .package(url: "https://github.com/pubnub/swift", exact: "8.0.0")
+    .package(url: "https://github.com/pubnub/kmp-chat", exact: "0.8.3-dev"),
```

Comment:
I assume the next tag for release would be `0.8.3-dev`. Right now, it points to the draft test release I created in `kmp-chat` to be able to test and submit this PR.

I'm not using `actions/cache@v4` in my workflow files to cache Swift Package Manager dependencies, as we used to do in the SDK. So, I hope that SPM will re-download the dependency if we create a "real" release at some point.